frontend/src/scenes/settings/project/SlackIntegration.tsx

```diff
@@ -49,79 +47,75 @@ export function SlackIntegration(): JSX.Element {
+                {slackIntegrations?.map((integration) => (
+                    <SlackIntegrationView
+                        key={integration.id}
+                        integration={integration}
+                        suffix={
+                            <LemonButton
+                                type="secondary"
+                                status="danger"
+                                onClick={() => onDeleteClick(integration.id)}
+                                icon={<IconTrash />}
+                            >
+                                Disconnect
                             </LemonButton>
-                        </>
-                    ) : (
-                        <>
-                            <h5>To get started</h5>
-                            <p>
-                                <ol>
-                                    <li>Copy the below Slack App Template</li>
-                                    <li>
-                                        Go to{' '}
-                                        <Link to="https://api.slack.com/apps" target="_blank">
-                                            Slack Apps
-                                        </Link>
-                                    </li>
-                                    <li>Create an App using the provided template</li>
-                                    <li>
-                                        <Link to={urls.instanceSettings()}>Go to Instance Settings</Link> and update the{' '}
-                                        <code>"SLACK_"</code> properties using the values from the{' '}
-                                        <b>App Credentials</b> section of your Slack Apps
-                                    </li>
-                                </ol>
+                        }
+                    />
+                ))}
 
-                                <CodeSnippet language={Language.JSON}>
-                                    {JSON.stringify(getSlackAppManifest(), null, 2)}
-                                </CodeSnippet>
-                            </p>
-                        </>
-                    )
-                ) : (
-                    <p className="text-muted">
-                        This PostHog instance is not configured for Slack. Please contact the instance owner to
-                        configure it.
-                    </p>
-                )}
+                <div>
+                    {addToSlackButtonUrl() ? (
+                        <Link to={addToSlackButtonUrl() || ''}>
+                            <img
+                                alt="Connect to Slack workspace"
+                                height="40"
+                                width="139"
+                                src="https://platform.slack-edge.com/img/add_to_slack.png"
+                                srcSet="https://platform.slack-edge.com/img/add_to_slack.png 1x, https://platform.slack-edge.com/img/add_to_slack@2x.png 2x"
```

Comment:
Due to self-hosting, airgaps, URLs changing, etc, we should just self-host these images.

Reply:
So last time I tried to get the slack app approved they would only do so if they could see were using their actual image 🤷